.github/workflows/main.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       # when changing the build matrix and changing the number of test runners
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9]
+        python: [3.5, 3.6, 3.7, 3.8, 3.9]
```

Comment:
you can also drop **3.5** support
https://devguide.python.org/#status-of-python-branches

Reply:
Yes, I know, I wanted to do this afterwards.